modules/040-node-manager/hooks/util.go

```diff
@@ -100,11 +100,19 @@ func setNodeGroupStatus(patcher *object_patch.PatchCollector, nodeGroupName stri
 	}
 	patchNodeGroupStatus(patcher, nodeGroupName, statusPatch)
 }
+func conditionsToPatch(c []ngv1.NodeGroupCondition) []map[string]interface{} {
+	b, _ := json.Marshal(c)
+	var r []map[string]interface{}
+	_ = json.Unmarshal(b, &r)
```

Comment:
ouch. It's a bit expensive operation. We don't have something like []byte structure here, only for controllable structures. Maybe we can do it manually. Or better add a method like `ToMap() map[string]interface{}` to the NodeGroupCondition. Really don't like marshaling/unmarshaling here %(

Reply:
done: ToMap() map[string]interface{} was made